my-dsl.md

```diff
@@ -1,38 +1,47 @@
 # Language
 _What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+
+Shepherd http://github.hubspot.com/shepherd/docs/welcome/
 
 
 # Domain
-_Describe the language's domain in five words._
+Web pages that require tutorials
 
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+Shepherd is a JavaScript library used to guide users through a web page. It allows informative textboxes to be easily pinned to elements through the use of id’s and class names. Users go through the tutorial sequentially through mouse clicks of two buttons: a “next” button that moves to the next step of the tutorial and either a “back” or “exit” button. It relies on another open-source library, Tether, to position all of its steps. 
```

Comment:
Not all libraries are DSLs, and not all DSLs are libraries. A way to start to distinguish them is to see if they actually provide a new langage, i.e. something like `some_object.should.be(10)` versus `assert(some_object == 10)`. The first is a language, the second is just a function. Libraries tend to just provide functions and objects; DSLs provide a new way to approach and reason about problems.